tests/src/Appender/RollingFileAppenderTest.cs

```diff
@@ -240,13 +252,52 @@ public void RollingCombinedWithPreserveExtension()
 			VerifyFileCount(2, true);
 		}
 
+		[Test]
+		public void RollingBySizePreserveExtensionAndWithFolder()
```

Comment:
`RollingBySizePreserveExtensionAndWithFolder` should probably read as `RollingBySizeAndPreserveExtensionShouldWorkWithSubdirectories` and have a doc string that explains further what the test is about.